.travis.yml

```diff
@@ -23,6 +23,7 @@ env:
     - SFDX_AUTOUPDATE_DISABLE=true
     - SFDX_USE_GENERIC_UNIX_KEYCHAIN=true
     - SFDX_DOMAIN_RETRY=300
+    - CODE_TESTS_WORKSPACE=$TRAVIS_BUILD_DIR/packages/salesforcedx-vscode-core/test/sampleWorkspace/
```

Comment:
According to the [VSCode testing docs](https://code.visualstudio.com/docs/extensions/testing-extensions) , we have to add this environment var so the vscode test module knows the path of the workspace to load. 

Should I move this to a more top-level directory since since all the packages should work with a workspace like this? Or should I point it to use sfdx-simple in system-tests?